### PR TITLE
Items backend suman

### PR DESCRIPTION
**Pull request by SumanMohanty20012987**

### CODE REVIEW

File: server/controllers/items.js [hunk 1]
```diff
@@ -62,8 +67,151 @@ export async function report(req, res) {
       imageUrl,
       uid
     );
+    await client.set('getItem', JSON.stringify(getItem));
```
Comment (manisaiprasad) [inline server/controllers/items.js]: getItem with key

File: server/controllers/items.js [hunk 2]
```diff
@@ -62,8 +67,151 @@ export async function report(req, res) {
...
     res.status(201).json(newItem);
   } catch (e) {
     res.status(400).json({ message: e });
   }
 }
+
+export async function getReportedItems(req, res) {
+  try {
+    const getItem = await getAllItems();
+    await client.set('getItem', JSON.stringify(getItem));
```
Comment (manisaiprasad) [inline server/controllers/items.js]: set item with key

Comment (manisaiprasad) [reply server/controllers/items.js]: can you check the redis code again

**Comment by SumanMohanty20012987** [comment]
redis done

**Review by manisaiprasad — APPROVED** [review]

**Pull request merged**